File: EduHubConnect-Backend-UsingSpringBoot/src/main/java/com/connect/eduHub/dto/FriendRequestDTO.java
package com.connect.eduHub.dto;

import java.time.LocalDateTime;

import com.connect.eduHub.model.RequestStatus;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

@Data
@AllArgsConstructor
@RequiredArgsConstructor
public class FriendRequestDTO {
	private Long id;
    private Long senderId;
    private Long receiverId;
    private RequestStatus status;
    private LocalDateTime sentAt;
    
    //added
    private String senderName;
    private String userBio;
    private String senderEmail;
    private String senderProfileImage;
}

File: EduHubConnect-AndroidFrontEnd/app/src/main/java/com/connect/eduhubconnect/dto/FriendRequestDTO.kt
package com.connect.eduhubconnect.dto

data class FriendRequestDTO(
    val id: Long,
    val senderId: Long,
    val receiverId: Long,
    val status: RequestStatus,
    val sentAt: String,
    val senderName : String,
    val userBio : String,
    val senderEmail:String,
    val senderProfileImage:String
)
